pennylane/drawer/representation_resolver.py

```diff
@@ -18,21 +18,25 @@
 import numpy as np
 import pennylane as qml
 
-from .charsets import UnicodeCharSet
+from .charsets import CHARSETS, UnicodeCharSet
 
 
 class RepresentationResolver:
     """Resolves the string representation of PennyLane objects.
 
     Args:
         charset (CharSet, optional): The CharSet to be used for representation resolution.
+        offsets (dict(string, int), optional): Offset the printed index of different symbol types in nested circuits.
```

Comment:
Minor suggestion: may be good to rename this argument to make it more descriptive: e.g., `custom_offsets` or `offset_per_unit` or something better :smile: .

Reply:
How about label_offsets?